sdks/python/README.md

```diff
@@ -16,6 +16,7 @@
     specific language governing permissions and limitations
     under the License.
 -->
+== This page is currently being updated. ==
```

Comment:
Did you mean `==` to render as something (like bold, header etc)?

Could you explain in a sentence what is being updated or link to the issue.